src/modules/autoAcceptRelationshipCreationChanges/AutoAcceptRelationshipCreationChangesModule.ts

```diff
@@ -39,10 +37,11 @@ export default class AutoAcceptRelationshipCreationChangesModule extends Connect
 
     private isIncomingPendingRelationshipCreationChange(event: RelationshipChangedEvent) {
         const data = event.data;
-        if (data.changes.length !== 1) return false;
+        if (data.status !== RelationshipStatus.Pending) return false;
 
-        const creationChange = event.data.changes[0];
-        return creationChange.request.createdBy !== this.currentIdentity && creationChange.status === RelationshipChangeStatus.Pending;
+        const creationChange = event.data.auditLog.find((log) => !log.oldStatus && log.newStatus === RelationshipStatus.Pending);
```

Comment:
I think we can massively simplify this.

1. there is only one entry here when the relationship is pending, so you can still look at index 0 (maybe checking for the length and return if its longer than expected)
2. please don't call it creationChange, instead just auditLogEntry
3. you can then just check for the createdBy for the entry on index 0 like you already do